### PR TITLE
first step towards image refactoring

### PR DESCRIPTION
**Pull request by glyph**
Begin to address #492 - this is just an initial example of how to use a plain-vanilla Image class rather than a subclass; there's lots more refactoring work to do, but this is a proof of concept.

[Review on Reviewable](https://reviewable.io/reviews/rackerlabs/mimic/495)

### CODE REVIEW

**Comment by glyph** [comment]
Hey @gabecase - we're pretty light on code reviewers right now, do you think you could have a look?

**Comment by gabecase** [comment]
@glyph sure I can look at this some tonight and tomorrow morning, will put some comments here on anything I find

**Comment by lekhajee** [comment]
+1 lgtm. Sorry it took so long.

**Comment by lekhajee** [comment]
@gabecase: feel free to merge the PR if you don't have any comments.

**Comment by gabecase** [comment]
Ok awesome 2 sets of eyes even better :) checking this out now

File: mimic/model/rackspace_images.py [hunk 1]
```diff
@@ -358,13 +404,13 @@ class RackspaceFedoraImage(Image):
...
     """
     images = {"Fedora 21 (PVHVM)": {"minRam": 512, "minDisk": 20,
-                                    "OS-EXT-IMG-SIZE:size": Image.image_size(),
+                                    "OS-EXT-IMG-SIZE:size": random_image_size(),
                                     "com.rackspace__1__ui_default_show": "True",
-                                    "id": Image.image_id()},
+                                    "id": new_random_image_id()},
               "Fedora 22 (PVHVM)": {"minRam": 512, "minDisk": 20,
-                                    "OS-EXT-IMG-SIZE:size": Image.image_size(),
+                                    "OS-EXT-IMG-SIZE:size": random_image_size(),
```
Comment (gabecase) [inline mimic/model/rackspace_images.py]: :+1: I like the re-useable random_image methods for the test data

**Comment by gabecase** [comment]
LGTM!

**Pull request merged**